dietpi/dietpi-firstboot

```diff
@@ -0,0 +1,215 @@
+#!/bin/bash
+{
+	#////////////////////////////////////
+	# DietPi FirstBoot Script
+	#
+	#////////////////////////////////////
+	# Created by Frederic Guilbault / fred@0464.ca
+	#
+	#////////////////////////////////////
+	#
+	# Info:
+	# - filename /DietPi/dietpi/dietpi-frostboot
+	# - activates on the first boot from dietpi-firstboot.service, runs before dietpi-boot.service and networking
+	#////////////////////////////////////
+
+	# Import DietPi-Globals --------------------------------------------------------------
+	/DietPi/dietpi/func/dietpi-obtain_hw_model # Runs every boot to allow e.g. switching SDcards between devices and to be failsafe
+	. /DietPi/dietpi/func/dietpi-globals
+	G_PROGRAM_NAME='DietPi-PreBoot'
```

Comment:
DietPi-FirstBoot

Reply:
fixed in 2cf3fad79ff1a3f7805c94d7d9f071661b5075a3